### PR TITLE
Running make upgrade on repo

### PR DESCRIPTION
**Pull request by jinder1s**

### CODE REVIEW

File: requirements/django.txt [hunk 1]
```diff
@@ -6,28 +6,26 @@
 #
 appdirs==1.4.3
 backports.os==0.1.1
-boto3==1.10.17            # via fs-s3fs
-botocore==1.13.17         # via boto3, s3transfer
+boto3==1.10.18            # via fs-s3fs
+botocore==1.13.18         # via boto3, s3transfer
 django-pyfs==2.0
 django==1.11.26
-docutils==0.15.2          # via botocore
```
Comment (jmbowman) [inline requirements/django.txt]: All of the dependencies of `botocore` vanished; try adding `--rebuild` to the pip-compile commands in `make upgrade` to see if that fixes it.

File: requirements/constraints.txt [hunk 2]
```diff
@@ -15,4 +15,4 @@ futures ; python_version == "2.7"
 
 # latest botocore release cap upper version of python-dateutil to 2.8.0 and that is not
 # compatible with other packages. That's why adding a cap here to avoid boto latest release untill they fix the issue.
```
Comment (jmbowman) [inline requirements/constraints.txt]: Should update the comment to reflect what's being pinned.  Preferably also link to https://github.com/boto/botocore/issues/1872 to make it easier to track when this is fixed upstream.

File: requirements/constraints.txt [hunk 3]
```diff
@@ -13,6 +13,8 @@
 futures ; python_version == "2.7"
 
 
-# latest botocore release cap upper version of python-dateutil to 2.8.0 and that is not
-# compatible with other packages. That's why adding a cap here to avoid boto latest release untill they fix the issue.
-botocore<1.14.0,>=1.13.17
+# latest botocore release cap upper version of python-dateutil<2.8.1, this conflicts
+# with our dependencies due to the order in which the dependecies are pip-compiled.
+# Adding this constraint till botocore can figure out a fix around this, keep a look at
+# https://github.com/edx/XBlock/pull/437
```
Comment (jmbowman) [inline requirements/constraints.txt]: Shouldn't this be a link to the botocore issue instead of this PR itself?

Comment (jinder1s) [reply requirements/constraints.txt]: lol yes, I seem to have grabbed the wrong one.

**Pull request merged**